metricflow-semantics/tests_metricflow_semantics/snapshots/test_semantic_model_container.py/tuple/test_linkable_elements_for_no_metrics_query__result0.txt

```diff
@@ -419,7 +419,6 @@
   'lux_listing__listing__lux_listing__twice_bookings_fill_nulls_with_0_without_time_spine',
   'lux_listing__listing__lux_listing__views',
   'lux_listing__listing__lux_listing__views_times_booking_value',
-  'metric_time__day',
```

Comment:
Wait, is this expected?

Reply:
Yes! I had the same question and meant to leave a comment explaining but I forgot.
This test is specifically filtering out specs with the property `DERIVED_TIME_GRANULARITY`. I'm not sure why. But I made a change in this PR that makes all `metric_time` granularities derived for no-metric queries.
Debatable which behavior would be expected for that property here, but doesn't seem super important since it's only internal.